layouts/_default/sitemap.xml

```diff
@@ -0,0 +1,28 @@
+{{/* This template is overridden to emit a versionless permalink for the most recent version. This allows us to submit the sitemap to crawlers with true canonical URLs. */}}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  {{ range .Data.Pages }}
+  <url>
+    {{- $permalink := .Permalink }}
+    {{- if ($.Site.Params.section.versions) }}
+      {{- if (eq (index $.Site.Params.section.versions 1) $.Site.Params.section.version) }}
```

Comment:
I took the approach of being least-invasive here -- only affecting the values for the current version, and leaving all other versions unaffected. This probably doesn't matter -- we aren't doing anything with older sitemaps anyway. If you feel like we should just always emit the versionless URL here, let me know!